src/components/ChangeData/ChangePassword.tsx

```diff
@@ -47,18 +47,18 @@ export const ChangePassword: React.FC<ChangePasswordProps> = ({
           id="password"
           type="text"
           label="Heslo"
+          className="dark"
```

Comment:
if you have light and dark, feel free to default to one of them, so the more used one can leave this prop out

Reply:
dark variant set as default